build.gradle

```diff
@@ -36,3 +39,12 @@ publishing {
         }
     }
 }
+
+// run google java format to enforce style
+spotless {
+    // uncomment this line to temporarily disable spotless (i.e. when debugging)
+    // enforceCheck = false
+    java {
+        googleJavaFormat()
```

Comment:
google-java-format produces [bad formatting](https://github.com/google/google-java-format/issues/5) for type annotations.
Therefore, I don't think its use should be encouraged, in projects related to type annotations, until https://github.com/google/google-java-format/issues/5 is fixed.

Reply:
Do you have a suggestion for a style checker that does not have bad formatting for type annotations? I'm not aware of one.

Reply:
https://github.com/plume-lib/run-google-java-format
It's not as convenient to use (ugly Gradle integration), but it formats correctly.